data/human/hails.txt

```diff
@@ -2794,6 +2797,454 @@ phrase "friendly pirate"
 	word
 		"."
 
+phrase "friendly pirate"
+	word
+		"We" 4
+		"We're short on supplies, so we"
+		"I haven't been on one for so long, so we"
+		"Come with us! We"
+		"You gotta come with us! We"
+	word
+		"'re going"
+		"'re about to go"
+		"'re planning to go"
+		" want to go"
+		"'re going to go"
+		"'re gonna go"
+	phrase
+		"pirate plans attack on planet 1"
+		"pirate plans attack on planet 2"
+		"pirate plans attack on ships"
+	word
+		"! "
+	phrase
+		"pirate attack gains"
+		"pirate enemy losses"
+	word
+		"!"
+
+phrase "pirate plans attack on planet 1"
+	word
+		" on a raid on "
+		" on an assault on "
+		" on an attack on "
+		" raiding "
+		" assaulting "
+		" attacking "
+	word
+		"the spaceport"
+		"the outfitter"
+		"the fuel depot"
+		"the weapons depot"
+		"some villages"
+		"some towns"
+		"a rival base"
+		"a rival post"
+		"a rival cell"
+		"a rival gang"
+		"a rival band"
+		"a competing base"
+		"a competing cell"
+		"a competing gang"
+		"a competing band"
+		"a base"
+		"a cell"
+		"a gang"
+		"a band"
+		"another gang"
+		"another band"
+		"another cell"
+		"our rivals"
+		"our competitors"
+		"some numbskulls"
+		"the Man"
+		"the Law"
+		"some of our friends"
+	word
+		" in "
+		" on "
+	phrase
+		"north pirate target"
+		"south pirate target"
+		"core pirate target"
+
+phrase "pirate plans attack on planet 2"
+	word
+		" on a raid on "
+		" on an assault on "
+		" on an attack on "
+		" raiding "
+		" assaulting "
+		" attacking "
+	phrase
+		"north pirate target"
+		"south pirate target"
+		"core pirate target"
+	word
+		"'s "
+	word
+		"spaceport"
+		"outfitter"
+		"fuel depot"
+		"weapons depot"
+		"villages"
+		"towns"
```

Comment:
We need to cover the case where the target name ends in an `s`, in which case the plural possesive is to just add an `'` instead of `'s`.
```suggestion
		"towns"
	replace
		"s's" "s'"
```